js/app/modules/card/knowledgeDocument.js

```diff
@@ -293,10 +297,18 @@ const KnowledgeDocument = new Module.Class({
 
             let [baseURI, hash] = decision.request.uri.split('#');
 
-            if (baseURI === this.model.ekn_id) {
+            if (baseURI.startsWith(this.model.ekn_id)) {
```

Comment:
Forgot to mention, but I didn't find a good way of doing this that doesn't require an external js lib.